### DISCUSSION TITLE
Crazyflie Propellors All Failing...

### DISCUSSION THREAD

**Discussion by akshaysunkara** [bitcraze/discussions#1429] [Q&A]
HEALTH: Acc noise floor variance X+Y:0.004766, (Z:0.004043)
HEALTH: Motor M1 variance X+Y: 11.64 (Z:4.16), voltage sag:0.42
HEALTH: Motor M2 variance X+Y: 176.63 (Z:87.75), voltage sag:0.43
HEALTH: Motor M3 variance X+Y: 10.19 (Z:10.77), voltage sag:0.43
HEALTH: Motor M4 variance X+Y: 32.02 (Z:31.22), voltage sag:0.43
HEALTH: Propeller test on M1 [FAIL]. low: 0.0, high: 2.50, measured: 11.64
HEALTH: Propeller test on M2 [FAIL]. low: 0.0, high: 2.50, measured: 176.63
HEALTH: Propeller test on M3 [FAIL]. low: 0.0, high: 2.50, measured: 10.19
HEALTH: Propeller test on M4 [FAIL]. low: 0.0, high: 2.50, measured: 32.02
HEALTH: Acc noise floor variance X+Y:0.005815, (Z:0.005134)
HEALTH: Motor M1 variance X+Y: 8.87 (Z:4.75), voltage sag:0.44
HEALTH: Motor M2 variance X+Y: 94.61 (Z:78.46), voltage sag:0.44
HEALTH: Motor M3 variance X+Y: 16.27 (Z:11.05), voltage sag:0.46
HEALTH: Motor M4 variance X+Y: 33.49 (Z:30.60), voltage sag:0.43
HEALTH: Propeller test on M1 [FAIL]. low: 0.0, high: 2.50, measured: 8.87
HEALTH: Propeller test on M2 [FAIL]. low: 0.0, high: 2.50, measured: 94.61
HEALTH: Propeller test on M3 [FAIL]. low: 0.0, high: 2.50, measured: 16.27
HEALTH: Propeller test on M4 [FAIL]. low: 0.0, high: 2.50, measured: 33.49

How can I fix these propellors failing? I bought these propellors recently: https://store.bitcraze.io/products/thrust-upgrade-bundle-for-crazyflie-2-x

I'm not able to fly it at all. 

The battery that I use is a different kind of battery with 600 mah, its too big for the frame provided in the crazyflie setup thing so i taped it onto the body. However, it doesnt affect the propelors at all. (Just wanted to give the little tidbit if that mightve affected the drone.)

**Comment by akshaysunkara** [comment]
STAB: WARNING: stabilizer loop rate is off (306)
HEALTH: Acc noise floor variance X+Y:0.007334, (Z:0.016181)
HEALTH: Motor M1 variance X+Y: 2.25 (Z:2.33), voltage sag:0.82
HEALTH: Motor M2 variance X+Y: 10.67 (Z:8.30), voltage sag:0.79
HEALTH: Motor M3 variance X+Y: 1.62 (Z:1.15), voltage sag:0.82
HEALTH: Motor M4 variance X+Y: 14.39 (Z:32.96), voltage sag:0.80
HEALTH: Propeller test on M2 [FAIL]. low: 0.0, high: 2.50, measured: 10.67
HEALTH: Propeller test on M4 [FAIL]. low: 0.0, high: 2.50, measured: 14.39
SUP: Can not fly
SUP: Ready to fly
SUP: Can not fly
SUP: Ready to fly
SUP: Can not fly
SUP: Ready to fly
HEALTH: Acc noise floor variance X+Y:0.006028, (Z:0.008132)
HEALTH: Motor M1 variance X+Y: 2.24 (Z:2.05), voltage sag:0.82
HEALTH: Motor M2 variance X+Y: 7.37 (Z:9.57), voltage sag:0.81
HEALTH: Motor M3 variance X+Y: 1.50 (Z:1.89), voltage sag:0.83
HEALTH: Motor M4 variance X+Y: 16.12 (Z:38.44), voltage sag:0.80
HEALTH: Propeller test on M2 [FAIL]. low: 0.0, high: 2.50, measured: 7.37
HEALTH: Propeller test on M4 [FAIL]. low: 0.0, high: 2.50, measured: 16.12


This is what happens after I used the old batteries and attached the frame properly. What's the issue here?

**Comment by akshaysunkara** [comment]
I changed the motors and propellors to new one, same kind and this is what I get:

HEALTH: Acc noise floor variance X+Y:0.004371, (Z:0.002685)
HEALTH: Motor M1 variance X+Y: 3.76 (Z:4.52), voltage sag:0.52
HEALTH: Motor M2 variance X+Y: 20.53 (Z:26.09), voltage sag:0.52
HEALTH: Motor M3 variance X+Y: 3.69 (Z:1.89), voltage sag:0.51
HEALTH: Motor M4 variance X+Y: 7.02 (Z:39.84), voltage sag:0.52
HEALTH: Propeller test on M1 [FAIL]. low: 0.0, high: 2.50, measured: 3.76
HEALTH: Propeller test on M2 [FAIL]. low: 0.0, high: 2.50, measured: 20.53
HEALTH: Propeller test on M3 [FAIL]. low: 0.0, high: 2.50, measured: 3.69
HEALTH: Propeller test on M4 [FAIL]. low: 0.0, high: 2.50, measured: 7.02

**Comment by akshaysunkara** [comment]
I kinda just want a stable experience while trying to fly. 

I even have a flowdeck if that makes a difference, haven't attached it yet.

**Comment by tobbeanton** [comment]
The propeller test will not work for the thrust-upgrade kit, it is not calibrated for this, it is calibrated for the stock-setup. You can still see from the values if any of the motors are showing excessive vibration. Then perhaps this motor or propeller needs to be replaced.
